<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>In2SGLClose</class>
 <widget class="QDialog" name="In2SGLClose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>In2 Printing Solutions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ATENCION! ESTA ACCION CERRARA EL TRABAJO ACTUAL</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="m_cbFinTrabajo">
       <property name="text">
        <string>Fin Trabajo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_cbFinBpadre">
       <property name="text">
        <string>Fin bulto padre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_cbFinBhijo">
       <property name="text">
        <string>Fin bulto hijo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_cbrechazo">
       <property name="text">
        <string>Fin Rechazo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>M.Impresos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lePrinted">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>M.Gastados</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_leUsed">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Comentario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="m_teComentario"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="m_cbSGLUpdate">
       <property name="text">
        <string>SGL update</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbOK">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
